jobHandler/run.py

```
@@ -0,0 +1,128 @@
+        await websocket.send(json.dumps(message))
+
+async def handle_response(message):   
+    '''
+        Handle response from worker and pass to client.
+    '''
+    global client, workers  
+    if client:
+        await send_to_client(client, message)
+    else:
+        message["response"] = "Client isn't initialized"
+        print(f"client isn't initialized")
```

Ruff (0.8.2)

f-string without any placeholders

Remove extraneous `f` prefix

(F541)